### ISSUE TITLE
Not getting good result after training

### ISSUE THREAD

**Issue by abdur4373** [iro-cp/FCRN-DepthPrediction#90]
Actually I have prepared my own data set of indoor scene in my environment and want to train model on that. I am freezing all other layers except for the up projection blocks and the result is not so good. Even I trained it on as small data set as 600 images and achieved 82 percent accuracy but the results were not good visually.
I donot know the reason of that maybe you can suggest me something.
And the images I want to train are approximately 6k. 
The pretrained weights with NYU are even performing better.
    batch_size = 32
    learning_rate = 1.0e-3
    monentum = 0.9
    weight_decay = 0.0005
    num_epochs = 70
 optimizer = torch.optim.SGD(filter(lambda p: p.requires_grad, model.parameters()), lr=learning_rate,
                                momentum=monentum, weight_decay=weight_decay)
and lr is halved after 10 epochs.

![MS_LAB_269_unfilled](https://user-images.githubusercontent.com/46248215/60284606-5fb44480-9925-11e9-9032-a6e4f90f9062.png)

Validation depth image
![Screenshot from 2019-06-27 21-37-28](https://user-images.githubusercontent.com/46248215/60284080-19121a80-9924-11e9-9c73-2689e3ed9831.png)

rgb image
![Screenshot from 2019-06-27 21-44-59](https://user-images.githubusercontent.com/46248215/60284489-1e239980-9925-11e9-9481-e6e043afafb7.png)

**Comment by pengzhi1998** [comment]
hi, did you solve the problem?

**Comment by mircial** [comment]
I have the same problem.  The result is very bad

**Comment by mircial** [comment]
> Actually I have prepared my own data set of indoor scene in my environment and want to train model on that. I am freezing all other layers except for the up projection blocks and the result is not so good. Even I trained it on as small data set as 600 images and achieved 82 percent accuracy but the results were not good visually.
> I donot know the reason of that maybe you can suggest me something.
> And the images I want to train are approximately 6k.
> The pretrained weights with NYU are even performing better.
> batch_size = 32
> learning_rate = 1.0e-3
> monentum = 0.9
> weight_decay = 0.0005
> num_epochs = 70
> optimizer = torch.optim.SGD(filter(lambda p: p.requires_grad, model.parameters()), lr=learning_rate,
> momentum=monentum, weight_decay=weight_decay)
> and lr is halved after 10 epochs.
> 
> ![MS_LAB_269_unfilled](https://user-images.githubusercontent.com/46248215/60284606-5fb44480-9925-11e9-9032-a6e4f90f9062.png)
> 
> Validation depth image
> ![Screenshot from 2019-06-27 21-37-28](https://user-images.githubusercontent.com/46248215/60284080-19121a80-9924-11e9-9c73-2689e3ed9831.png)
> 
> rgb image
> ![Screenshot from 2019-06-27 21-44-59](https://user-images.githubusercontent.com/46248215/60284489-1e239980-9925-11e9-9481-e6e043afafb7.png)

I have solved this problem. The bad result is due to dropout, so frcn methon is useful without dropout!

**Comment by mircial** [comment]
@abdur4373  The trainning may be useful without dropout operation in fcrn methon.
